modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -215,7 +215,9 @@ public function enqueue_scripts() {
 			'wpcomGutenberg',
 			array(
 				'switchToClassic' => array(
-					'isVisible' => false,
+					'isVisible' => $is_calypsoify,
+					'label'     => __( 'Switch to Classic Editor', 'jetpack' ),
+					'url'       => Jetpack_Calypsoify::getInstance()->get_switch_to_classic_editor_url(),
 				),
 				'isCalypsoify'    => $is_calypsoify,
```

Comment:
I forgot to remove this property on https://github.com/Automattic/jetpack/pull/12070, so would you mind to remove it here? With https://github.com/Automattic/wp-calypso/pull/32421 changes, `wpcomGutenberg.isCalypsoify` is no longer needed.

Reply:
7d2ee54f2